koji/flatten-el8/koji-ansible/roles/grobisplitter/files/rhel8-split.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
I'm always a fan of `-e`
```suggestion
#!/bin/bash -e
```

Reply:
. I'd prefer not to deviate from their script more than necessary at this time.